bugbug/bug_features.py

```diff
@@ -87,11 +87,14 @@ class whiteboard(object):
     def __call__(self, bug):
         ret = []
 
-        # TODO: Add any [XXX:YYY] that appears in the whiteboard as [XXX: only
-
-        for elem in ['memshrink', '[ux]']:
-            if elem in bug['whiteboard'].lower():
-                ret.append(elem)
+        for whiteboard_word in bug['whiteboard'].lower().split(']'):
```

Comment:
You can use `re` to split by multiple characters at the same time. Split also by space, as otherwise something like `whiteboard1 whiteboard2` won't work.

Also add more cases to the test, creating "fake" bugs with only values in the `whiteboard` field.
- `whiteboard1 whiteboard2`
- `[whiteboard1]whiteboard2`
- `[whiteboard1] whiteboard2`
- `whiteboard1[whiteboard2]`
- `whiteboard1 [whiteboard2]`

Reply:
> You can use re to split by multiple characters at the same time.

In particular, `re.split`.

Reply:
Splitting at space has a drawback though: In cases like `[AV:Quick Heal][regressed sept 6th][dll version is 3.0.1.*]`.